test/box-luatest/gh_8817_bug_set_language_shared_test.lua

```diff
@@ -0,0 +1,84 @@
+local t = require('luatest')
```

Comment:
Test passed on tarantool without fix.

Reply:
Done, the problem is reproduced only if the server is raised via `box.cfg{listen=...}`, not via `console.listen(...)`. For server with `console.listen(...)` there is no such bug